beginner_source/dcgan_faces_tutorial.py

```diff
@@ -265,7 +265,7 @@
 plt.axis("off")
 plt.title("Training Images")
 plt.imshow(np.transpose(vutils.make_grid(real_batch[0].to(device)[:64], padding=2, normalize=True).cpu(),(1,2,0)))
-
+plt.show()
```

Comment:
My guess is that you are running this as a Python script? `plt.show()` is not necessary in notebooks which was the primary targeted environment for this tutorial. We can add this as a comment.
```suggestion
# If running as a Python script, uncomment this line to display the image: 
# plt.show()
```

Reply:
Actually, let's just add it.